ci/docker/Dockerfile.build.jetson

```diff
@@ -39,6 +47,9 @@ RUN git clone --recursive -b v0.2.20 https://github.com/xianyi/OpenBLAS.git && \
     make -j$(nproc) && \
     PREFIX=${CROSS_ROOT} make install
 
+ENV OpenBLAS_HOME=${CROSS_ROOT}
+ENV OpenBLAS_DIR=${CROSS_ROOT}
```

Comment:
Unrelated change?

Reply:
I also don't understand this one, can we add a comment on what do those variables do?

Reply:
This is important for [FindOpenBLAS.cmake](https://github.com/apache/incubator-mxnet/blob/master/cmake/Modules/FindOpenBLAS.cmake#L49-L52) it is not used here because it's still a makefile build, but will be used when the build will be ported.